File: remove_avatars.py
import os
import re

# Files to process
files_to_process = [
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(cliente)\perfil\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(cliente)\meus-agendamentos\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(cliente)\feedback\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(cliente)\dashboard\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(cliente)\agendar\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(profissional)\profissional\perfil\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(profissional)\profissional\dashboard\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\profissionais\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\feedbacks\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\dashboard\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\comissoes\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\bloqueios\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\assinantes\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(admin)\admin\agenda\page.tsx",
    r"c:\Users\MARCIO.PETIGROSSO\barbeariadomoura-1\src\app\(public)\sobre\profissionais\page.tsx",
]

for filepath in files_to_process:
    if not os.path.exists(filepath):
        print(f"File not found: {filepath}")
        continue
    
    with open(filepath, 'r', encoding='utf-8') as f:
        lines = f.readlines()
    
    new_lines = []
    skip_next = False
    for i, line in enumerate(lines):
        # Skip lines that contain <AvatarImage
        if '<AvatarImage' in line:
            continue
        new_lines.append(line)
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.writelines(new_lines)
    
    print(f"Processed: {filepath}")

print("Done!")

File: remove_avatars.js
const fs = require('fs');
const path = require('path');

const files = [
    'src/app/(cliente)/perfil/page.tsx',
    'src/app/(cliente)/meus-agendamentos/page.tsx',
    'src/app/(cliente)/feedback/page.tsx',
    'src/app/(cliente)/dashboard/page.tsx',
    'src/app/(cliente)/agendar/page.tsx',
    'src/app/(profissional)/profissional/perfil/page.tsx',
    'src/app/(profissional)/profissional/dashboard/page.tsx',
    'src/app/(admin)/admin/profissionais/page.tsx',
    'src/app/(admin)/admin/feedbacks/page.tsx',
    'src/app/(admin)/admin/dashboard/page.tsx',
    'src/app/(admin)/admin/comissoes/page.tsx',
    'src/app/(admin)/admin/bloqueios/page.tsx',
    'src/app/(admin)/admin/assinantes/page.tsx',
    'src/app/(admin)/admin/agenda/page.tsx',
    'src/app/(public)/sobre/profissionais/page.tsx',
];

files.forEach(file => {
    const filePath = path.join(__dirname, file);

    if (!fs.existsSync(filePath)) {
        console.log(`File not found: ${file}`);
        return;
    }

    const content = fs.readFileSync(filePath, 'utf8');
    const lines = content.split(/\r?\n/);
    const newLines = lines.filter(line => !line.includes('<AvatarImage'));

    fs.writeFileSync(filePath, newLines.join('\r\n'), 'utf8');
    console.log(`Processed: ${file}`);
});

console.log('Done!');
